### PR TITLE
Add documentation about the new users api endpoint

### PR DESCRIPTION
**Pull request by jenskueper**

### CODE REVIEW

File: source/developers-guide/rest-api/api-resource-user/index.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+## Introduction
+
+This chapter of the documentation is about the API's user resource.
+With this resource, it's possible to retrieve, update, create and
+delete backend user of your shop. 
+
+
+For each scenario, we provide an example of the data
+which is required from, as well as an exemplary response.
```
Comment (mitelg) [inline source/developers-guide/rest-api/api-resource-user/index.md]: the from sound wrong here

File: source/developers-guide/rest-api/api-resource-user/index.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+            "extendedEditor": false,
+            "disabledCache": false
+        }
+    ],
+    "success": true
+}
+```
+
+Attention: The properties apiKey, sessionId and password are missing,
+the API user neither has the update nor the create privilege.
```
Comment (mitelg) [inline source/developers-guide/rest-api/api-resource-user/index.md]: is there an `if` missing before `the API`?

File: source/developers-guide/rest-api/api-resource-user/index.md (hunk 1 above)
Comment (jenskueper) [reply source/developers-guide/rest-api/api-resource-user/index.md]: Fixed it

File: source/developers-guide/rest-api/api-resource-user/index.md (hunk 2 above)
Comment (jenskueper) [reply source/developers-guide/rest-api/api-resource-user/index.md]: You are right! Fixed it.

**Comment by mitelg** [comment]
docs for https://github.com/shopware/shopware/pull/1090

**Pull request merged**